subgraphs/spotify/src/utils/mocks.ts

```diff
@@ -322,14 +323,14 @@ export class MockedSpotifyDataSource implements SpotifyDataSource {
     id: string,
     params?: { additional_types?: string; fields?: string; market?: string }
   ): Promise<Spotify.Object.Playlist> {
-    throw new Error('Method not implemented.');
+    return mocks.playlists.find((t) => t.id == id);
```

Comment:
```suggestion
    return mocks.playlists.find((t) => t.id === id);
```

Reply:
For completeness, should this `throw` if the playlist can't be found? If so, we might want to check the error message used from the "real" API and mimic that.